src/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -15,11 +16,11 @@ namespace System
     public abstract class Delegate : ICloneable, ISerializable
     {
         // _target is the object we will invoke on
-        internal object _target;
+        internal object _target = null!; // Initialized by VM as needed
```

Comment:
Shouldn't this be `object?` ?, this should be null if the target is static

Reply:
Yes, it should be `object?`.